adapters/algorix/algorix.go

```diff
@@ -135,7 +135,7 @@ func preProcess(request *openrtb2.BidRequest) {
 			if err != nil {
 				continue
 			}
-			if impExt.Prebid != nil && impExt.Prebid.IsRewardedInventory == 1 {
+			if impExt.Prebid != nil && impExt.Prebid.IsRewardedInventory != nil && *impExt.Prebid.IsRewardedInventory == 1 {
```

Comment:
I changed IsRewardedInventory to a pointer since we now need to marshal the imp.ext.Prebid object and it wasn't designed for that originally. You'll see similar changes with omitempty and some effected adapter json tests.